modules/admin/ref-rbac-permission-policies.adoc

```diff
@@ -118,4 +133,44 @@ Kubernetes permissions::
 |
 |
 |Allows user or role to access the proxy endpoint
-|===
\ No newline at end of file
+|===
+
+OCM permissions::
+
+[cols="15%,25%,15%,45%", frame="all", options="header"]
+|===
+|Name
+|Resource type
+|Policy
+|Description
+
+|`ocm.entity.read`
+|
+|read
+|Allows user or role to read from the OCM plugin
+
+|`ocm.cluster.read`
+|
+|read
+|Allows user or role to read the cluster information in the OCM plugin
+|===
+
+Topology permissions::
+
+[cols="15%,25%,15%,45%", frame="all", options="header"]
+|===
+|Name
+|Resource type
+|Policy
+|Description
+
+|`topology.view.read`
+|
+|read
+|Allows user or role to view the topology plugin
+
+|`kubernetes.proxy`
+|
+|
+|Allows user or role to access the proxy endpoint (ability to read pod logs and events within Showcase and {product-very-short})
```

Comment:
Showcase is our upstream demo platform, and as such, we do not include it in our enterprise documentation.
```suggestion
|Allows user or role to access the proxy endpoint, allowing them to read pod logs and events within {product-very-short}.
```